vrx_ign/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ install(
 
 # Plugins
 list(APPEND VRX_IGN_PLUGINS
```

Comment:
There may be a better way to do this, but when I built the Stationkeeping plugin I had to pull the base ScoringPlugin out into a separate library to get everything to link correctly.  I treated it just like the Waves library (but with the right dependencies) and was then able to add Stationkeeping to the for loop with no issues.

Reply:
It makes sense that we have to do that. I updated the code in [ef2115b](https://github.com/osrf/vrx/pull/501/commits/ef2115b7bc8933cacb57dd8061ba472159c968f1).